eitprocessing/datahandling/mixins/slicing.py

```diff
@@ -149,7 +149,10 @@ def select_by_time(  # noqa: D417
 
     @property
     def t(self) -> TimeIndexer:
-        """Time Indexer property."""  # TODO: add a short explanation of what this is used for.
+        """Time indexer.
+
+        See TimeIndexer.
```

Comment:
This is not very useful as a docstring, from the user interface perspective. (It's fine for developers who have it right beneath in the same file.
I often think it would be great if you could make docstrings f-strings to add identical passages in multiple, but unfortunately this doesn't work.

Reply:
Is there a way to easily link to another docstring? I guess that is useful to the online documentation (Sphinx), but not IDE documentation.

I copied the doc from TimeIndexer in f615c0c53c46b21826f59192074dbc5fab0c77bb.

Reply:
I mean, ideally it would also work for the IDE documentation. I actually don't know whether it works in sphinx to pass an f-string and import a piece of text from elsewhere, but for IDE documentation (at least in VS code), I looked into it a little while ago for another project, but didn't find a way to do it. 
I can ask around at the office, or maybe stackoverflow.